ao_stock/models/stock_move.py

```diff
@@ -5,7 +5,7 @@
 
 
 class StockMoveLine(models.Model):
-    _inherit = "stock.move.line"
+    _inherit = "stock.move"
```

Comment:
```python
class StockMove
```

Reply:
what can we do now? :scream_cat::scream_cat:

Reply:
another PR